src/OpenTelemetry.AutoInstrumentation.Native/util.cpp

```diff
@@ -102,6 +103,30 @@ std::vector<WSTRING> GetEnvironmentValues(const WSTRING& name)
     return GetEnvironmentValues(name, L',');
 }
 
+std::vector<WSTRING> GetEnabledEnvironmentValues(const bool enabled_by_default, const std::unordered_map<WSTRING, WSTRING> values_map)
+{
+    std::vector<WSTRING> values;
+ 
+    for (auto value : values_map)
+    {
+        bool enabled = enabled_by_default;
+        const auto env_value = GetEnvironmentValue(value.second);
+        if (env_value == WStr("true"))
```

Comment:
AFAIK everywhere else we are checking also for `1`

I think it not compliant with OTel spec, but I prefer to have consistency. Probably there is some macro for this check.

Reply:
We should drop support for 0 and 1. Will be handled in scope of #2089